staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/devicestatus.go

```diff
@@ -49,15 +51,22 @@ func (deviceStates *DeviceStates) PushStatesToEdgeCore() {
 		DeviceNamespace: deviceStates.DeviceNamespace,
 	}
 
-	log.Printf("send statesRequest", statesRequest.DeviceName, statesRequest.State)
-	if err := grpcclient.ReportDeviceStates(statesRequest); err != nil {
+	klog.V(4).Infof("send device %s status %s request to cloud", statesRequest.DeviceName, statesRequest.State)
+	if err = grpcclient.ReportDeviceStates(statesRequest); err != nil {
 		klog.Errorf("fail to report device states of %s with err: %+v", deviceStates.DeviceName, err)
 	}
 }
 
 func (deviceStates *DeviceStates) Run(ctx context.Context) {
-	// TODO setting states reportCycle
-	ticker := time.NewTicker(2 * time.Second)
+	// No need to report device status to the cloud
+	if !deviceStates.ReportToCloud {
+		return
+	}
+	// Set device status report cycle
+	if deviceStates.ReportCycle == 0 {
+		deviceStates.ReportCycle = common.DefaultReportCycle
+	}
+	ticker := time.NewTicker(deviceStates.ReportCycle)
```

Comment:
Define a minimum unit of time. `deviceStates.ReportCycle*time.Second` or `deviceStates.ReportCycle*time.Millisecond`

Reply:
We have defined the time unit during initialization
![image](https://github.com/user-attachments/assets/81758ff1-ba66-40da-ab01-0041dca9d5ad)